eventhubsprocessor/checkpoint.py

```diff
@@ -0,0 +1,19 @@
+"""
+Author: Aaron (Ari) Bornstien
+"""
+class Checkpoint:
+    """
+    Contains checkpoint metadata
+    """
+    def __init__(self, partition_id, offset="-1", sequence_number="0"):
```

Comment:
I believe we keep it for information in case it's useful the processing function. This is what the c# client did so I copied their practice. Happy to remove if you think that that is worthwhile.

Reply:
Let's keep it. It is an integer so set the default to integer 0 rather than a string.